app/src/main/res/layout/main_fragment.xml

```diff
@@ -112,9 +113,10 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="32dp"
-            android:text="@string/current_orders"
+            android:text="@{`Current orders(` + model.currentOrders + `)`}"
```

Comment:
Hardcoded string value

Reply:
Also looks as null on UI
![image](https://user-images.githubusercontent.com/13312041/130046066-285d377f-8487-460d-a11f-1e83e57540db.png)

Reply:
Are you sure your branch is updated? This problem is already fixed.